pkg/api/common.go

```diff
@@ -2,12 +2,15 @@ package api
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"math"
 	"net/http"
 	"regexp"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/pkg/errors"
```

Comment:
We use this at line `252`. However, this adds to the build size as `fmt` package is enough for this. Lets get rid with this import.